### PR TITLE
add CAUV can pmu and hv pm

### PR DESCRIPTION
**Pull request by cuhome**
Add hv pm and can pmu docs and add cuav to the hardware vendor

### CODE REVIEW

File: common/source/docs/common-hv-pm.rst [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+  * voltage detection accuracy: +-0.1v
+  * current detection accuracy: +-0.4A
+* Bec (5v) max current: 5A
+* Max (detection) current: 60A
+* Max output current(ESC/MOTOR PORT): 60A
+
+Enable HV PM
+============
+
+.. image:: ../../../images/hv_pm.png
```
Comment (tridge) [inline common/source/docs/common-hv-pm.rst]: this image link is not correctly formatted, it needs a target

File: common/source/docs/common-hv-pm.rst [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+* Monitor：Anglog Voltage and Current
+* Sensor：CUAV HV PM
+* APM Ver：CUAV v5 or APM1(V5 series flight control selects CUAV v5)
```
Comment (tridge) [inline common/source/docs/common-hv-pm.rst]: what do you mean by "APM1" here?

File: common/source/docs/common-powermodule-landingpage.rst [hunk 3]
```diff
@@ -19,6 +19,8 @@ The links below have information about the most commonly used power modules
     Power Module Configuration <common-power-module-configuration-in-mission-planner>
     Smart Batteries <common-smart-battery>
     Fuel Monitors <common-fuel-sensors>
+    Can PMU<common-can-pmu>
```
Comment (tridge) [inline common/source/docs/common-powermodule-landingpage.rst]: I think "UAVCAN PMU" would be better

File: common/source/docs/common-can-pmu.rst [hunk 4]
```diff
@@ -0,0 +1,98 @@
...
+* Connect the flight control CAN1/2 and the module CAN interface.
+* Connect the V5 series power cable to the V5 Flight Control Power2 (if other flight controllers are connect to the Power interface) and the module Power  interface.
+
+Enable CAN PMU
+==============
+
+Set the following parameters in the Mission planner's full parameter list and restart after writing:
+* CAN\_P1\_DRIVER=1 
+* CAN\_P2\_DRIVER=1 
+* BATT\_MONITOR=8（If using monitor 1）
```
Comment (tridge) [inline common/source/docs/common-can-pmu.rst]: what do you mean by "if using monitor 1" ?

File: common/source/docs/common-hv-pm.rst (hunk 1 above)
Comment (cuhome) [reply common/source/docs/common-hv-pm.rst]: Yes, but its name is can pmu

File: common/source/docs/common-can-pmu.rst (hunk 4 above)
Comment (cuhome) [reply common/source/docs/common-can-pmu.rst]: Battery monitor 1

**Comment by cuhome** [comment]
@tridge Already modified, please help me check it out.

File: common/source/docs/common-can-pmu.rst [hunk 5]
```diff
@@ -0,0 +1,100 @@
...
+* Connect the flight control CAN1/2 and the module CAN interface.
+* Connect the V5 series power cable to the V5 Flight Control Power2 (if other flight controllers are connect to the Power interface) and the module Power  interface.
+
+Enable CAN PMU
+==============
+
+Set the following parameters in the Mission planner's full parameter list and restart after writing:
+* CAN\_P1\_DRIVER=1 
+* CAN\_P2\_DRIVER=1 
+* BATT\_MONITOR=8（If using Battery monitor 1）
```
Comment (Hwurzburg) [inline common/source/docs/common-can-pmu.rst]: params should be hot linked in this form : :ref:\`CAN_P1_DRIVER<CAN_P1_DRIVER>\` = 1

File: common/source/docs/common-hv-pm.rst [hunk 6]
```diff
@@ -0,0 +1,30 @@
...
+* Max (detection) current: 60A
+* Max output current(ESC/MOTOR PORT): 60A
+
+Enable HV PM
+============
+
+.. image:: ../../../images/hv-pm.jpg
+:target: ../_images/hv-pm.jpg
+
+* Monitor：Anglog Voltage and Current
```
Comment (Hwurzburg) [inline common/source/docs/common-hv-pm.rst]: Should provide params as above in previous uavcan pmu, not just rely on Mission Planner to set them

File: common/source/docs/common-hv-pm.rst (hunk 1 above)
Comment (Hwurzburg) [reply common/source/docs/common-hv-pm.rst]: just fyi, actually I dont think a :target: is required....its the images clickable hotlink...if not there, its just an embedded image....almost all the ones existing are merely links to themselves and provide no new info...unlike say a partners logo which hyperlinks to the partners website....

File: common/source/docs/common-hv-pm.rst (hunk 6 above)
Comment (cuhome) [reply common/source/docs/common-hv-pm.rst]: Modified has been completed, please help me check it.

**Comment by Hwurzburg** [comment]
see 2234

**Pull request closed without merging**